first-sajal-verma

sajalverma17 opened the pull request:
I pulled the latest changes and did another commit ("Get latest") after my first commit "Add Sajal Verma to Contributors list" into my branch.

Is that okay?

sajalverma17 committed:
Add Sajal Verma to Contributors list

sajalverma17 committed:
Get latest

sara-02 commented on Contributors.md:
```diff
@@ -15116,6 +15116,10 @@ Christian Reyes
 - [Mervin Ng](https://github.com/m6ng)
 - [run me](https://github.com/run-me)
 - [Erez Ben Said](https://github.com/erezbens)
+<<<<<<< HEAD
```
Hey @sajalverma17 You followed the process correctly. However, once the conflicts are removed the <<<HEAD and ==== <<<Master and the  >>>>123...
Lines should also be removed from the file where the conflicts were resolved.

sara-02 commented:
@sajalverma17 Can you please update your branch again to resolve conflicts and also remove the unrequired lines post-conflict-resolution.

sara-02 commented on Contributors.md:
```diff
@@ -15116,6 +15116,10 @@ Christian Reyes
+<<<<<<< HEAD
+- [Sajal Verma] (https://github.com/sajalverma17)
+- [Arpit Pandey] (https://github.com./arpitansu)
+=======
```
Remove this line

sajalverma17 replied:
Removed the conflict markers

sara-02 commented on Contributors.md:
```diff
@@ -15195,3 +15199,4 @@ Christian Reyes
 - [Josh Carroll](https://github.com/joshc133)
 - [Henish Patel](https://github.com/henishpatel)
 - [Remie Malik] (https://github.com/remiemalik)
+>>>>>>> a9b6b250507baa1f469e499c4f7c22028af592a0
```
And this as well.

sajalverma17 replied:
Removed this extra line as well.

sajalverma17 committed:
Resolved conflicts and removed conflict markers

sajalverma17 committed:
Removed duplicated Remie Malik

sajalverma17 commented:
@sara-02, updated the pull request. 
Committed latest changes and removed the conflict markers. Removed one user's duplicated entry introduced due to my "Resolved conflicts and removed conflict markers" commit.

Can you please review and let me know if I did something wrong?
Thanks!

sara-02 commented on Contributors.md:
```diff
@@ -15149,6 +15149,8 @@ Christian Reyes
 - [Mervin Ng](https://github.com/m6ng)
 - [run me](https://github.com/run-me)
 - [Erez Ben Said](https://github.com/erezbens)
+- [Sajal Verma] (https://github.com/sajalverma17)
```
IF we put space between [] and(), then the hyperlink will not be rendered properly. 
`[Sajal Verma](https://github.com/sajalverma17)` how it should be.

sajalverma17 replied:
Sure! On it. Will update soon.

sara-02 reviewed (changes requested):
You resolved rest of the issues correctly, just a minor change since you are on it. Please fix the space between []() for your name.

sajalverma17 committed:
Removed space from name and URL

sajalverma17 commented:
Hi @sara-02
Added another commit to the branch to remove the space.
Thanks for the corrections.
